src/components/ViewControlMenu.tsx

```diff
@@ -0,0 +1,66 @@
+import { reportRejection } from 'lib/trap'
+import {
+  ContextMenu,
+  ContextMenuDivider,
+  ContextMenuItem,
+  ContextMenuItemRefresh,
+  ContextMenuProps,
+} from './ContextMenu'
+import { AxisNames, VIEW_NAMES_SEMANTIC } from 'lib/constants'
+import { useModelingContext } from 'hooks/useModelingContext'
+import { useMemo } from 'react'
+import { sceneInfra } from 'lib/singletons'
+
+export function useViewControlMenuItems() {
```

Comment:
Heh, interesting pattern. I would have simply made this a component.

Reply:
Yeah I think this is kinda whack. In the future I or someone should rework the `ContextMenu` component to just take children instead of an array of `items`